quickstarts/about-quickstarts.md

```diff
@@ -1,37 +1,107 @@
-# The quickstart tutorial template
 
-The quickstart or `Hello World!` tutorial is for introducing your users to your product or framework for the first time. Users can typically complete these tutorials in less than 15 minutes. It's task-based and installation-focused.
+# Quickstart Guide
 
-## About the overview section
+{Read this document before you start working on the [quickstart template](template-quickstarts.md)}
 
-Use this section to provide a short description of your product and its purpose, and to describe what the user will accomplish in this tutorial.
+## Introduction
 
-You can also state the intended audience for your tutorial and mention any concepts users should be familiar with before starting the tutorial. 
+The quickstart is intended for introducing your users to your product or framework for the first time. 
+It focuses on the **hero feature** of the product and helps your users to start using the product as quickly as possible.
```

Comment:
I'm unfamiliar with the term "hero feature." Could you explain what it means? Is it essentially the main feature of the product?

Reply:
Nevermind, I answered my own question as I reread the template. I might suggest using "main" here and putting hero in parentheses (the same way you did in the Tips section).

Reply:
We have replaced the word Hero with Primary.